tests/tutorials/test_excel_tutorial.py

```diff
@@ -0,0 +1,202 @@
+import xlwings as xw
+import unittest
+from datetime import datetime
+import pytz
+
+import lusid
+from lusid import models
+from lusid.utilities.api_client_builder import ApiClientBuilder
+from utilities.credentials_source import CredentialsSource
+from utilities.data_source import DataSource
+
+
+def get_date(date, sht):
+    date_at = sht.range(date).value
+    if date_at:
+        date_at = datetime(date_at.year, date_at.month, date_at.day, tzinfo=pytz.utc)
+    return date_at
+
+
+class Scopes(unittest.TestCase):
+    # This test validates that the data displayed in "LUSID Excel -Setting up your IBOR template Global Wquity 
+    # Fund.xlsx" is correct using the python SDK. 
+
+    @classmethod
+    def setUpClass(cls):
+        api_client = ApiClientBuilder().build(CredentialsSource.secrets_path())
+
+        cls.scopes_api = lusid.ScopesApi(api_client)
+        cls.portfolios_api = lusid.PortfoliosApi(api_client)
+        cls.transaction_portfolios_api = lusid.TransactionPortfoliosApi(api_client)
+
+    def test_list_scopes(self):
+
+        # define location of data in excel spreadsheet
+        book_name = "LUSID Excel - Setting up your IBOR template Global Equity Fund.xlsx"
+        book_path = DataSource.data_path(book_name)
+        sheet_name = 'View scopes'
+        range_containing_scopes = 'D16:D26'
+
+        # get scopes from excel
+        wb = xw.Book(book_path)
+        self.assertTrue(wb, msg="Error loading book: " + book_name)
+
+        sht = wb.sheets(sheet_name)
+        self.assertTrue(sht, msg="Error loading sheet: " + sheet_name)
+
+        scopes = sht.range(range_containing_scopes).value
+        self.assertTrue(scopes, msg="Error loading scopes from range: " + range_containing_scopes)
+
+        excel_scopes = []
+        scopes = scopes[0:9]
+        for scope in scopes:
+            if scope:
```

Comment:
can `scope` ever be `None`? If it is I think you can replace this with something like

`excel_scopes = [scope for scope in scope[0:9] if scope is not None]`

Reply:
Equity changed